contracts/Market.sol

```diff
@@ -93,7 +100,7 @@ contract Market is
   /**
    * @notice The BridgedPolygonNORI contract.
    */
-  BridgedPolygonNORI private _bridgedPolygonNORI;
+  IERC20WithPermit private _bridgedPolygonNORI;
```

Comment:
Looking ahead to the possible USDC work I'm tempted to rename this variable here and in `RestrictedNORI`.